x/bank/migrations/v047/gen_state.go

```diff
@@ -0,0 +1,14 @@
+package v047
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+// MigrateGenState accepts exported v0.43 x/bank genesis state and migrates it to
+// v0.47 x/bank genesis state. The migration includes:
+// - Move the SendEnabled entries from Params to the new GenesisState.SendEnabled field.
+func MigrateGenState(oldState *types.GenesisState) *types.GenesisState {
```

Comment:
Double check that this is still needed.

Reply:
Yup. Still needed. Used by the genutil migration stuff. It could probably have been just replaced with calling `MigrateSendEnabled()` on the oldState, but it's probably modeled after other migration stuff. Oh well. Since that's what went into the Cosmos-SDK, we'll do the same here. Keep it simple.